modules/terraform/validate_struct.go

```diff
@@ -0,0 +1,99 @@
+package terraform
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/collections"
+)
+
+type ValidationOptions struct {
+	RootDir string
+	//Sub-directories relative to the RootDir to walk recursively for all Terraform modules. For example, if your
+	// RootDir is /home/project and you want to validate all modules in home/project/terraform then pass "terraform"
+	// in TargetSubDirs
+	TargetSubDirs []string
+	//ExcludeDirs must be relative to the RootDir. For example if your RootDir is home/project and you want to exclude
+	// the path /home/project/test, pass "test" in ExcludeDirs
```

Comment:
It looks like this has to be the full path in the struct, but need to be relative in `NewValidationOptions`. In that case, this doc should mention that it needs to be full path, and the `NewValidationOptions` function will take relative paths and turn it into full paths.